src/popup/router/pages/Dashboard.vue

```diff
@@ -160,10 +165,11 @@ export default {
 @use '../../../styles/typography';
 @use '../../../styles/mixins';
 
-.account {
+.dashboard {
   display: flex;
   flex-direction: column;
   min-height: 100%;
+  background-color: variables.$color-bg-3-new;
```

Comment:
This is not necessary. This background is set when the route has the `newUI` meta set to true. But in general I fixed the BG behaviour here: https://github.com/aeternity/superhero-wallet/pull/1555

Reply:
![image](https://user-images.githubusercontent.com/9008074/189301293-7721b562-5c04-461b-ae6f-ee92469d9a83.png)